docs/integration/js-example.md

```diff
@@ -1,9 +1,154 @@
 ---
-sidebar_position: 11
+sidebar_position: 8
 ---
 
 # Javascript Example Game Integration
 
-We have an example game of TicTacToe implemented in TypeScript with NodeJS.
+In the next section we'll guide you through the relevant changes that you have to make to your game.
+
+We have an example game of TicTacToe implemented in TypeScript with NodeJS. You can get it from [github.com/UltimateTournament/NodeJS-Example-TicTacToe](https://github.com/UltimateTournament/NodeJS-Example-TicTacToe). You can also check out [the PR](https://github.com/UltimateTournament/NodeJS-Example-TicTacToe/pull/2) which contains most of these changes.
+
+Note that the example game is a "pool" game (more on that below) but we'll list the options for all game modes and note when each is applicable.
+
+## Integrating the game into the Ultimate Arcade
+
+### Dockerfile for your server
```

Comment:
Let's point them to the one in the example?

Reply:
Why not to the one included in the SDK? I expect that we'll forget to update the example after a while but hopefully not the SDK

Reply:
Fair